### PR TITLE
Tolerations support added

### PR DESCRIPTION
**Pull request by wowsel**
Hello! 
This MR include tolerations support for your operator. 
Tested and working in my k8s 1.11.7

### CODE REVIEW

File: api/redisfailover/v1alpha2/types.go [hunk 1]
```diff
@@ -31,6 +31,7 @@ type RedisFailoverSpec struct {
 	// NodeAffinity defines the rules for scheduling the Redis and Sentinel
 	// nodes
 	NodeAffinity *corev1.NodeAffinity `json:"nodeAffinity,omitempty"`
+	Tolerations []corev1.Toleration `json:"tolerations,omitempty"`
```
Comment (jchanam) [inline api/redisfailover/v1alpha2/types.go]: Could you add a comment about this new parameter of the spec?

Comment (wowsel) [reply api/redisfailover/v1alpha2/types.go]: Tolerations provides a way to schedule Pods on Tainted Nodes

Comment (jchanam) [reply api/redisfailover/v1alpha2/types.go]: I meant inside the code, so everyone can read it (like the rest of parameters)

Comment (wowsel) [reply api/redisfailover/v1alpha2/types.go]: Sure, done. Sorry

**Pull request merged**